File: AutoevaluacionWeb/src/main/java/co/com/poli/autoevaluacion/services/ParametrosServices.java
package co.com.poli.autoevaluacion.services;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import co.com.poli.autoevaluacion.domain.Parametros;
import co.com.poli.autoevaluacion.repositories.ParametrosRepository;

@Service
public class ParametrosServices {

	@Autowired
	private ParametrosRepository parametrosRepository;
	
	public void guardarParametro(Parametros parametros){
		parametrosRepository.save(parametros);
	}
	
	public List<Parametros> listarParametros(){
		return parametrosRepository.findAllByOrderByParametrosID();
	}
	
	public void eliminarParametro(Long parametroID){
		parametrosRepository.delete(parametroID);
	}
}

File: AutoevaluacionWeb/src/main/resources/static/service/parametrosServices.js
angular.module("myApp")
.service("parametrosServices" , ["$http", function($http){
	var dataService = {};
	
	dataService.guardarParametro = function (parametro){
		return $http.post('/parametros/guardar', parametro);
	}
	
	dataService.listarParametros = function (){
		return $http.get('/parametros/listar');
	}
	
	return dataService;
}]);